tap_dbt/streams.py

```diff
@@ -69,11 +73,15 @@ def get_url_params(
         """Return offset as the next page token."""
         params = {}
         _ = context
+        params["limit"] = self.config.get("page_size", 100)
```

Comment:
@aaronsteers, @edgarrmondragon - I'm not familiar enough with the SDK to know if `page_size` could be set in get_new_paginator and then retrieved from there instead of going to config again to pick it up, feels a bit like duplication of code. It works, but perhaps the SDK has a better way of doing this.